pkg/virt-controller/watch/application.go

```diff
@@ -1012,6 +1015,9 @@ func (vca *VirtControllerApp) setupLeaderElector() (err error) {
 			Callbacks: leaderelection.LeaderCallbacks{
 				OnStartedLeading: vca.onStartedLeading(),
 				OnStoppedLeading: func() {
+					if err := metrics.UnregisterLeaderMetrics(); err != nil {
+						golog.Fatal(err)
```

Comment:
nit: since we are already fatal exiting in the line below, I think we can just log the error with quitting here